PublicMapping/districtbuilder
# Add Project Evaluate view for contiguity

## Overview

Adds project evaluate mode view for contiguity.

- Add Contiguity detail
- Display contiguity on map
- Add map legend for contiguity

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/112548477-b072fe80-8d92-11eb-8fc1-397b9e3469ed.png)

## Testing Instructions

- ./scripts/server
- Open a project and click evaluate
- Click 'Contiguity'
- Expect: Contiguity view is rendered as pictured

Closes #605
